src/views/HousingLottery/studentView/index.jsx

```diff
@@ -41,7 +46,13 @@ const StudentView = () => {
 
   const handleClick = async () => {
     try {
-      console.log(application_id);
+      let application_id = nanoid(8),
+        timeTarget = new Date(dueDate + ' 11:24:00 PM').getTime(),
```

Comment:
Is there a specific reason the time '11:24:00 PM' has been linked to timeTarget?